lockbox-ios/Store/AccountStore.swift

```diff
@@ -142,7 +142,7 @@ extension AccountStore {
     }
 
     private func clear() {
-        KeychainWrapper.wipeKeychain()
+        keychainWrapper.clearAllValues(for: .account)
```

Comment:
IIUC This is the point which fixes the bug.

Reply:
Yes 👍